tests/extensions/test_classification.py

```diff
@@ -0,0 +1,302 @@
+                ],
+            )
+        ]
+    )
+
+    plain_item.validate()
+    assert (
+        ClassificationExtension.ext(plain_item).bitfields is not None
+        and len(cast(List[Bitfield], ClassificationExtension.ext(plain_item).bitfields))
+        == 1
+    )
+    assert (
+        cast(List[Bitfield], ClassificationExtension.ext(plain_item).bitfields)[
+            0
+        ].offset
+        == 0
+    )
+    assert (
+        cast(List[Bitfield], ClassificationExtension.ext(plain_item).bitfields)[
+            0
+        ].length
+        == 1
+    )
+    assert (
+        ClassificationExtension.ext(plain_item).bitfields is not None
+        and len(
+            cast(List[Bitfield], ClassificationExtension.ext(plain_item).bitfields)[
+                0
+            ].classes
+        )
+        == 2
+    )
+
+
+def test_create_classes(plain_item: Item) -> None:
+    ClassificationExtension.add_to(plain_item)
+    ext = ClassificationExtension.ext(plain_item)
+    ext.apply(
+        bitfields=[
+            Bitfield.create(
+                offset=0,
+                length=1,
+                classes=[
+                    Classification.create(description="no", value=0),
+                    Classification.create(description="yes", value=1),
+                ],
+            )
+        ]
+    )
+    ext.classes = [
+        Classification.create(description="no", value=0),
+        Classification.create(description="yes", value=1),
+    ]
+    assert ext.bitfields is None
+    ext.bitfields = [
+        Bitfield.create(
+            offset=0,
+            length=1,
+            classes=[
+                Classification.create(description="no", value=0),
+                Classification.create(description="yes", value=1),
+            ],
+        )
+    ]
+    assert ext.classes is None
+
+
+def test_create() -> None:
+    field = Bitfield.create(
+        name="cloud_confidence",
+        description="Cloud confidence levels",
+        offset=8,
+        length=2,
+        classes=[
+            Classification.create(
+                name="not_set", description="No confidence level set", value=0
+            ),
+            Classification.create(
+                name="low", description="Low confidence cloud", value=1
+            ),
+            Classification.create(
+                name="medium", description="Medium confidence cloud", value=2
+            ),
+            Classification.create(
+                name="high", description="High confidence cloud", value=3
+            ),
+        ],
+    )
+
+    assert (
+        field.__repr__() == "<Bitfield offset=8 length=2 classes=["
+        "<Classification value=0 description=No confidence level set>, "
+        "<Classification value=1 description=Low confidence cloud>, "
+        "<Classification value=2 description=Medium confidence cloud>, "
+        "<Classification value=3 description=High confidence cloud>]>"
+    )
```

Comment:
Can you remove this assertion? Testing `__repr__` is an anti-pattern for me, since its only for human consumption (not really part of our software API).

Reply:
`__repr__` testing was pulled in from [test for EO](https://github.com/stac-utils/pystac/blob/main/tests/extensions/test_eo.py#L36).  But it did feel dirty.  Happy to remove the assert.  Will preserve the call to maintain coverage.